.godir

```diff
@@ -1 +1 @@
-github.com/skygeario/skygear-server
+github.com/inizio/skygear-server
```

Comment:
This file is unnecessary when we use go modules. Feel free to delete anything blocking you, for example the `default.nix`.